planning/behavior_velocity_planner/src/scene_module/run_out/scene.cpp

```diff
@@ -305,12 +307,19 @@ std::vector<geometry_msgs::msg::Point> RunOutModule::createVehiclePolygon(
 {
   const float base_to_rear = planner_param_.vehicle_param.base_to_rear;
   const float base_to_front = planner_param_.vehicle_param.base_to_front;
-  const float half_width = planner_param_.vehicle_param.width / 2.0;
-
-  const auto p1 = tier4_autoware_utils::calcOffsetPose(base_pose, base_to_front, half_width, 0.0);
-  const auto p2 = tier4_autoware_utils::calcOffsetPose(base_pose, base_to_front, -half_width, 0.0);
-  const auto p3 = tier4_autoware_utils::calcOffsetPose(base_pose, -base_to_rear, -half_width, 0.0);
-  const auto p4 = tier4_autoware_utils::calcOffsetPose(base_pose, -base_to_rear, half_width, 0.0);
+  const double base_to_right =
+    (planner_param_.vehicle_param.wheel_tread / 2.0) + planner_param_.vehicle_param.right_overhang;
+  const double base_to_left =
+    (planner_param_.vehicle_param.wheel_tread / 2.0) + planner_param_.vehicle_param.left_overhang;
+
+  //  const float half_width = planner_param_.vehicle_param.width / 2.0;
```

Comment:
please remove this comment

Reply:
I will check unnecessary comments. Thank you